src/odemis/acq/test/millmng_test.py

```diff
@@ -108,7 +108,8 @@ def setUpClass(cls):
 
         cls.acq_streams = [fs1]
 
-        current_stage_pos = cls.stage.position.value
+        # set the current stage position same as GRID 1 for milling
+        current_stage_pos = cls.stage.getMetadata()[model.MD_FAV_POS_ACTIVE]
```

Comment:
Note that currently that's not GRID 1, it's not even on top of a grid. Not that the simulator really cares though... I'll send separately a PR which changes the microscope file to have the FAV_POS_ACTIVE really containing to the center of GRID 1. That's how it should be.